docs/issues/inspector.md

```
@@ -0,0 +1,15 @@
+# Inspector gizmo (🔍)
+
+Labels: inspector, ui, devtools, project:newton
+Milestone: PROJECT: Newton
+
+## Summary
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)